scripts/check.ts

```diff
@@ -0,0 +1,208 @@
+import { CurveYieldStrategy__factory, vaults } from '@ragetrade/sdk';
```

Comment:
We should either remove this file if we are using it for informal checking or rename it so that its purpose is clear.

Reply:
Update: protodev is moving this to a test case.